libraries/AP_SmartAudio/AP_SmartAudio.h

```diff
@@ -0,0 +1,267 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#ifndef HAL_SMARTAUDIO_ENABLED
+#define HAL_SMARTAUDIO_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
Is it possible to warn the user if they try to enable smartaudio but it isn't included in the build? It is often obvious to the user that some feature isn't included because relevant parameters don't exist. But in this case, all the VTX_* parameters will exist because they are also used for crossfire.

Reply:
The obvious thing to do is remove the serial protocol number - but we don't do that for any other protocol, so what is there now is at least consistent :)